### PR TITLE
Add swift framework testing

### PR DESCRIPTION
**Pull request by granluo**

### CODE REVIEW

**Comment by granluo** [comment]
Swift test for storage was done in 7d71d1e.

File: .github/workflows/storage.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ jobs:
 
   catalyst:
     # Don't run on private repo unless it is a PR.
-    if: github.repository != 'FirebasePrivate/firebase-ios-sdk' || github.event_name == 'pull_request'
+    # if: github.repository != 'FirebasePrivate/firebase-ios-sdk' || github.event_name == 'pull_request'
```
Comment (paulb777) [inline .github/workflows/storage.yml:47]: Undo comments here and below

File: scripts/setup_quickstart_framework.sh [hunk 2]
```diff
@@ -25,12 +27,18 @@ mkdir -p Firebase/
...
+if [ ! -f "Firebase/module.modulemap" ]; then
+  mv "${HOME}"/ios_frameworks/Firebase/module.modulemap Firebase/
+fi
 for file in "$@"
 do
+  if [ ! -f "Firebase/${file}" ]; then
   # Not override framework if a framework with the same name was moved early..
-  mv -n ${file} Firebase/
+    mv -n ${file} Firebase/
+	fi
```
Comment (paulb777) [inline scripts/setup_quickstart_framework.sh:41]: indent

File: scripts/setup_quickstart_framework.sh [hunk 3]
```diff
@@ -15,7 +15,9 @@
 set -ex
 
 REPO=`pwd`
-git clone https://github.com/firebase/quickstart-ios.git
+if [ ! -d "quickstart-ios" ]; then
+				git clone https://github.com/firebase/quickstart-ios.git
```
Comment (paulb777) [inline scripts/setup_quickstart_framework.sh:19]: fix indent

**Pull request merged**